code/ui/manager/src/components/hooks/useShowToolbar.tsx

```diff
@@ -0,0 +1,25 @@
+import { useContext } from 'react';
+import { ManagerContext, addons } from '@storybook/manager-api';
+
+const isFunction = (val: unknown): val is CallableFunction => typeof val === 'function';
+
+const useShowToolbar = (showToolbar: boolean) => {
+  const { state } = useContext(ManagerContext);
+
+  const config = addons.getConfig();
+  if (!isFunction(config?.toolbar?.showToolbar)) {
+    return showToolbar;
+  }
+
+  const params = {
+    path: state.path,
+    viewMode: state.viewMode,
+    singleStory: state.singleStory,
+    layout: state.layout,
+    storyId: state.storyId,
+  };
+
+  return config.toolbar.showToolbar(params);
```

Comment:
Can we pass the whole entry in here so you can read tags etc?

Reply:
Sure. Is there a way I can record that the toolbar doc will need updating once tags are out, then?

Reply:
@kylegach is there a good place to track this sort of thing?